src/offliner.js

```diff
@@ -228,6 +228,9 @@
       case 'xpromise':
         this._receiveCrossPromise(msg.id, msg.order);
         break;
+      case 'checkForActivationPending':
```

Comment:
I think the correct wording (here and elsewhere) is `check for pending activation`, but I'm not totally sure :smile: